code/modules/clothing/accessories/holomap.dm

```diff
@@ -34,6 +34,25 @@ var/list/holomap_cache = list()
 	holomap_color = "#13B40B"
 
 
+/obj/item/clothing/accessory/holomap_chip/ert
+	name = "emergency response team holomap chip"
+	icon_state = "holochip_ert"
+	marker_prefix = "ert"
+	holomap_filter = HOLOMAP_FILTER_ERT
+	holomap_color = "#5FFF28"
+
+	var/list/prefix_update = list(
```

Comment:
You can use absolutely any data type except numbers and `null` for the key of a dictionary.

Reason it doesn't work here is because instead of being sane and using a special syntax, they decided to use a special construct function which means you can't do non-strings from a simple `list()` call.